remote_syslog.go

```diff
@@ -53,6 +54,11 @@ func tailOne(file string, excludePatterns []*regexp.Regexp, logger *syslog.Logge
 // Tails files speficied in the globs and re-evaluates the globs
 // at the specified interval
 func tailFiles(globs []string, excludedFiles []*regexp.Regexp, excludePatterns []*regexp.Regexp, interval RefreshInterval, logger *syslog.Logger, severity syslog.Priority, facility syslog.Priority, poll bool) {
+	if len(globs) == 0 {
```

Comment:
Can this ever evaluate to `true` given that we're checking [here](https://github.com/featalion/remote_syslog2/blob/fix-issue-43/config_manager.go#L41) and this is invoked via the call made [early on in main](https://github.com/featalion/remote_syslog2/blob/fix-issue-43/remote_syslog.go#L113).

Also, could this message appear on the console when r_s2 is damonized?